src/fu-bluez-device.c

```diff
@@ -0,0 +1,304 @@
+/*
+ * Copyright (C) 2021 Ricardo Cañuelo <ricardo.canuelo@collabora.com>
+ *
+ * SPDX-License-Identifier: LGPL-2.1+
+ */
+
+#define G_LOG_DOMAIN				"FuBluezDevice"
+
+#include "config.h"
+
+#include <string.h>
+
+#include "fu-bluez-device.h"
+#include "fu-common.h"
+#include "fu-device-private.h"
+#include "fu-firmware-common.h"
+
+#define DEFAULT_PROXY_TIMEOUT	5000
+
+/**
+ * SECTION:fu-bluez-device
+ * @short_description: a Bluez Bluetooth LE device
+ *
+ * An object that represents a Bluez Bluetooth LE device.
+ *
+ * See also: #FuBleDevice
+ */
+
+struct _FuBluezDevice {
+	FuBleDevice		 parent_instance;
+	GHashTable		*uuid_paths;	/* utf8 : utf8 */
+};
+
+G_DEFINE_TYPE (FuBluezDevice, fu_bluez_device, FU_TYPE_BLE_DEVICE)
+
+#define GET_PRIVATE(o) (fu_bluez_device_get_instance_private (o))
+
+/**
+ * fu_bluez_device_add_uuid_path:
+ * @self: A #FuBluezDevice
+ * @uuid: (nullable): The UUID, e.g. `FIXME:better-example-please`
+ * @uuid: (nullable): The path, e.g. `FIXME:better-example-please`
```

Comment:
Just a thought to prevent us missing fixing these in CI - I wonder if we want a grep for `FIXME` in `git diff` failing a job

Reply:
I'm not so sure. Wed just physiologically just switch to `XXX`, `TODO` or something else.

Reply:
Assuming this only applies to `master`, how about using this thing: https://probot.github.io/apps/todo/ with the keywords `FIXME` and `TODO`

Reply:
Not convinced dude; too many false positives in the existing code. Plus we use FIXME in one of the metainfo files to avoid people uploading it as-is to the LVFS...